frontend/src/metabase/lib/cookies.js

```diff
@@ -31,6 +32,29 @@ var MetabaseCookies = {
         } catch (e) {
             console.error("setSessionCookie:", e);
         }
+    },
+
+    setHasSeenAlertSplash: (hasSeen) => {
+        const options = {
+            path: window.MetabaseRoot || '/',
+            expires: 365,
```

Comment:
This is more of a complaint with `js-cookie` than with your code but it's very confusing to see a cookie being set and then seeing the `expires` field not be a date. My first instinct was that this cookie was being set with bad values or "cooked incorrectly" if you will. IRL `Expires` is supposed to be a date https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies. 

So I guess maybe it's nice this library is trying to be magical but I just find it confusing. Any chance we can get a comment saying "yes, this is correct" so I don't flip out next time I run into this?